src/edu/csus/ecs/pc2/ui/TestResultsFrame.java

```diff
@@ -102,5 +102,18 @@ public void setValidatorOutputFileNames(String[] filenames) {
         getMultiTestSetOutputViewerPane().setValidatorOutputFileNames(filenames);
     }
 
+    /**
+     * Enables/Disables Execute All button
+    */
+    public void setExecuteAllButton(boolean b) {
```

Comment:
As the COMMENT above says, this method is enabling/disabling the button.  However, the _name_ of the method doesn't say anything about "enabling"; it talks about "setting".  It's not completely clear what it means to "set" a button.  It could mean "position it in a specific location"; it could mean "set the text on the button"; it could mean other things.  

I think the method should be named (something like) `setExecuteAllButtonEnabled(boolean)`, or perhaps `enableExecuteAllButton(boolean)`.

Reply:
Updated